src/Api/appsettings.json

```diff
@@ -20,6 +20,14 @@
     "Playfab": {
       "PlayfabTitleId": "titleId",
       "PlayfabSecretKey": "secretKey"
+    },
+    "QuestProvider": {
+      "Ygg": {
+        "Endpoint": "https://test-api2.yieldguild.io/api",
+        "ApiKey": "test_api_key",
+        "quests": [
+        ]

```

Comment:
Where is the quest definition atm ?

Reply:
We don't have any official quest yet. We had one they did for testing purposes, but this one doesn't consider any metric that we actually support.

Reply:
Why not downloading a json when server boots ?

Reply:
Because getting the config from EnvVar was already tested, I just wanted to push it forward to focus on finishing the sync.

I'll be doing it anytime soon, I liked you idea when we talked about it.